cves/CVE-2010-3118.yml

```
@@ -135,12 +173,20 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer: 
+  answer: |
+    I saw some commits that referenced changes being made in order to support
+    or better support ChromeOS, which had not yet been released.
+
+    The initial vulnerability was introduced many years ago but due to its minor
+    nature, the vulnerability was only realized by developers when it was announced
+    publicly during a talk.
   events:
-  - name: 
-    date: 
-  - name: 
-    date: 
+    event:
+    - name: Original Chormium Bug 1647 Logged. See https://bugs.chromium.org/p/chromium/issues/detail?id=1647
```

Is this meant to say "Original Chromium Bug"?
```suggestion
    - name: Original Chromium Bug 1647 Logged. See https://bugs.chromium.org/p/chromium/issues/detail?id=1647
```